Gopkg.toml

```diff
@@ -68,6 +68,10 @@
   name = "github.com/segmentio/analytics-go"
   version = "2.1.1"
 
+[[constraint]]
```

Comment:
Can we remove this unnecessary constraint?

Reply:
👌